Makefile

```diff
@@ -79,115 +84,51 @@ test:
 	go test -timeout ${TEST_TIMEOUT} -mod=vendor ${TEST_FLAGS} ./...
 
 ################################################################################
-# maistra v2.0
+# Helm charts generation
 ################################################################################
-.PHONY: update-remote-maistra-2.0
-update-remote-maistra-2.0:
+.PHONY: update-remote-maistra-%
+update-remote-maistra-%:
+	$(eval version:=$(subst update-remote-maistra-,,$@))
```

Comment:
perhaps you can use `$*` for this purpose (applies everywhere is this pr)

Reply:
@jwendell Done in 25845f47